### PR TITLE
feat: [CCM-9035]: Adding governance feature

### PR DESCRIPTION
**Pull request by sahibaCCM**
## Describe your changes
Added Governance in the feature list as a part of onboarding AWS Connector for cloud asset governance in CCM

## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/38419)

### CODE REVIEW

**Comment by sahibaCCM** [comment]
trigger authorcheck

**Comment by sahibaCCM** [comment]
trigger ti1
trigger codebasehash
trigger messagecheck

**Comment by sahibaCCM** [comment]
trigger messagecheck

File: 440-connector-nextgen/src/test/resources/gcpccmconnector/gcpCcmConfig.json [hunk 1]
```diff
@@ -24,4 +24,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (sdsingh07) [inline 440-connector-nextgen/src/test/resources/gcpccmconnector/gcpCcmConfig.json:27]: Can we undo this change? As we haven't added any changes in this file.

Comment (sahibaCCM) [reply 440-connector-nextgen/src/test/resources/gcpccmconnector/gcpCcmConfig.json]: Done

**Review by sdsingh07 — DISMISSED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, 2 of 3 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38419#-:-NDhGvK1DawpCQOfK-Df:bnfp4nl)* status: :shipit: complete! ✓

**Review by nikunj-badjatya — APPROVED** [review]
> Reviewed 1 of 3 files at r2, 2 of 5 files at r4, 4 of 4 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38419)* status: :shipit: complete! ✓

**Review by hitesharinga — APPROVED** [review]
> Reviewed 1 of 3 files at r2, 1 of 1 files at r3, 2 of 5 files at r4, 4 of 4 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38419)* status: :shipit: complete! ✓
